frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -123,7 +129,26 @@ export function MainNavbarView({
   const ONBOARDING_URL = "/getting-started";
   const isNewInstance = useSelector(getIsNewInstance);
   const canAccessOnboarding = useSelector(getCanAccessOnboardingPage);
-  const showOnboardingLink = isNewInstance && canAccessOnboarding;
+  const showOnboardingLink =
+    !isOnboardingLinkDismissed && isNewInstance && canAccessOnboarding;
+  const isOnboardingPageSelected = nonEntityItem?.url === ONBOARDING_URL;
+
+  const dispatch = useDispatch();
+
+  const dismissOnboardingLink = () => {
+    setIsOnboardingLinkDismissed(true);
+
+    if (isOnboardingPageSelected) {
+      dispatch(push("/"));
+    }
+
+    dispatch(
+      addUndo({
+        icon: "gear",
+        message: <OnboardingDismissedToast />,
+      }),
```

Comment:
Given the number of words in this toast, I think it might help to make it display for twice as long

Reply:
I think you can do:

```ts
   addUndo({
        icon: "gear",
        message: <OnboardingDismissedToast />,
        timeout: 10000,
      }),
```

Reply:
Goes against the product doc/design decision.